reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -321,278 +310,308 @@ class StaticHtmlReporter : Reporter() {
-
-                    row.scanErrors.forEach { id, errors ->
-                        append("""
-                                <a href="#$id">$id</a>
-                                <ul>
-                                ${errors.joinToString("\n") {
-                                    "    <li><p>${it.description.replace("\n", "</br>")}</p>" +
-                                            "<p>${it.resolutionDescription}</p></li>"
-                                }}
-                                </ul>""".trimIndent())
-                    }
+                tbody { evaluatorErrors.forEach { evaluatorRow(it) } }
+            }
+        }
+    }
 
-                    append("""
-                            </td>
-                        </tr>""".trimIndent())
-                }
+    private fun TBODY.evaluatorRow(error: ResolvableIssue) {
+        val cssClass = when {
+            error.isResolved -> "ort-resolved"
+            else -> "ort-error"
+        }
+
+        tr(cssClass) {
+            td { +error.source }
+            td {
+                p { +error.description }
+                p { +error.resolutionDescription }
+            }
+        }
+    }
+
+    private fun DIV.errorTable(errorSummary: ErrorTable) {
+        h2 {
+            id = "error-summary"
+            +"Error Summary (${errorSummary.rows.count()})"
+        }
 
-                append("</tbody></table>")
+        p { +"Errors from excluded components are not shown in this summary." }
+
+        h3 { +"Packages" }
+
+        table("ort-report-table ort-errors") {
+            thead {
+                tr {
+                    th { +"Package" }
+                    th { +"Analyzer Errors" }
+                    th { +"Scanner Errors" }
+                }
             }
 
-    private fun createTable(title: String, vcsInfo: VcsInfo?, summary: ProjectTable, anchor: String) =
-            buildString {
-                val excludedClass = if (summary.exclude != null) " ort-excluded" else ""
+            tbody { errorSummary.rows.forEach { errorRow(it) } }
+        }
+    }
+
+    private fun TBODY.errorRow(row: ReportTableModel.ErrorRow) {
+        tr("ort-error") {
+            td { +"${row.id}" }
 
-                append("<h2 id=\"$anchor\">$title</h2>")
+            td {
+                row.analyzerErrors.forEach { id, errors ->
+                    a("#$id") { +"$id" }
 
-                summary.exclude?.let { exclude ->
-                    append("""
-                        <h3>Project is Excluded</h3>
-                        <p>
-                            The project is excluded for the following reason:<br/>
-                            <div class="reason">${exclude.reason} - ${exclude.comment}</div>
-                        </p>""".trimIndent())
+                    ul {
+                        errors.forEach { error ->
+                            li {
+                                p { +error.description.replace("\n", "<br/>") }
```

Comment:
In the final HTML, this seems to cause `<br/>` to be written as `&lt;br/&gt;`.

Reply:
Thanks, fixed by wrapping with `unsafe {}`.